modules/imgproc/include/opencv2/imgproc.hpp

```diff
@@ -4081,6 +4081,13 @@ CV_EXPORTS Ptr<GeneralizedHoughBallard> createGeneralizedHoughBallard();
 CV_EXPORTS Ptr<GeneralizedHoughGuil> createGeneralizedHoughGuil();
 
 //! Performs linear blending of two images
```

Comment:
Usually the `dst` computation formula is here, not at the end if the description.